### PR TITLE
Create short reg service route: apps.host.crt-stage.com

### PR DESCRIPTION
**Pull request by alexeykazakov**

### CODE REVIEW

File: setup_cluster.sh [hunk 1]
```diff
@@ -83,6 +83,7 @@ function deploy_operators() {
     NAME=host-operator OPERATOR_NAME=toolchain-host-operator NAMESPACE=$HOST_OPERATOR_NS envsubst < $PREVIOUS_DIR/config/operator_deploy.yaml | oc apply -f -
     make get-registration-service-repo login-as-admin
     make deploy-registration HOST_NS=$HOST_OPERATOR_NS REG_IMAGE_NAME=quay.io/codeready-toolchain/registration-service:v0.1
+    oc apply -f $PREVIOUS_DIR/config/reg_service_route.yaml
```
Comment (xcoulon) [inline setup_cluster.sh:86]: what about making this call to `oc apply` after the `cd $PREVIOUS_DIR` below?

**Review by MatousJobanek — APPROVED** [review]

File: setup_cluster.sh (hunk 1 above)
Comment (alexeykazakov) [reply setup_cluster.sh:86]: It doesn't really change anything, does it?

Comment (xcoulon) [reply setup_cluster.sh:86]: no, it just seems more logical to do the `oc apply` after the `cd` since this is where the file is located

Comment (xcoulon) [reply setup_cluster.sh:86]: but I will approve the PR as-is, it's was just a minor comment

**Review by xcoulon — APPROVED** [review]

File: setup_cluster.sh (hunk 1 above)
Comment (alexeykazakov) [reply setup_cluster.sh:86]: yeah maybe it would look slightly better..

**Pull request merged**